### PR TITLE
Add Playlist Features

### PR DESCRIPTION
**Pull request by Shailseen**
**Features to be reviewed**
- Modal
- Playlist Services
  - `Add video to playlist` Video-Library/my-video/src/services/PlaylistServices/addVideoToPlaylistService
  - `Create playlist` Video-Library/my-video/src/services/PlaylistServices/createPlaylistService
  - `Delete playlist` Video-Library/my-video/src/services/PlaylistServices/deletePlaylistService
  - `Delete video from playlist` Video-Library/my-video/src/services/PlaylistServices/deleteVideoPlaylistService
  - `Get playlist video` Video-Library/my-video/src/services/PlaylistServices/getPlaylistVideosService

[Preview Link](https://deploy-preview-5--gaari-dekho.netlify.app/)

### CODE REVIEW

File: src/components/Card/CreatePlaylistCard/CreatePlaylistCard.module.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  cursor: pointer;
+}
+.btn_disabled {
+  cursor: not-allowed;
+  background-color: grey;
+  color: white;
+  border: 1px solid white;
+}
+.btn_disabled:hover {
+  background-color: grey;
```
Comment (maitrakhatri) [inline src/components/Card/CreatePlaylistCard/CreatePlaylistCard.module.css:28]: use hex code for colours

**Review by maitrakhatri — APPROVED** [review]
Clean code, easy to read and understand. 

Suggestions: On the website, when I try to do anything without logging in, it shows "You have to login first" but it is in white colored toast and bg is also white, so it is not catching my attention. I confused it with "this feature is not working". So I'll suggest either make it a red alert or place it a the center of the screen or you can directly redirect the user to log in page

File: src/routes/PlaylistVideos/PlaylistVideos.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+//   playlistVideos.videos.map((item) => {
+//     return (
+//       <HorizontalCard
+//         key={item._id}
+//         playlistId={playlistId}
+//         cardData={item}
+//         type="playlist"
+//       />
+//     );
+//   })}
```
Comment (surbhi84) [inline src/routes/PlaylistVideos/PlaylistVideos.jsx]: can be removed if not required.

**Review by surbhi84 — APPROVED** [review]
- The functionality is working.
- Code is  easily readable.
- Looks good to merge.
suggestion-do look into into absolute paths, its easy to use and will make your code look a lot more clean.

**Comment by Shailseen** [comment]
> Clean code, easy to read and understand.
> 
> Suggestions: On the website, when I try to do anything without logging in, it shows "You have to login first" but it is in white colored toast and bg is also white, so it is not catching my attention. I confused it with "this feature is not working". So I'll suggest either make it a red alert or place it a the center of the screen or you can directly redirect the user to log in page

i will do that later

File: src/routes/PlaylistVideos/PlaylistVideos.jsx (hunk 2 above)
Comment (Shailseen) [reply src/routes/PlaylistVideos/PlaylistVideos.jsx]: yes, i forget it

**Pull request merged**